app/main.ipynb

```diff
@@ -408,10 +525,38 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 24,
-   "outputs": [],
+   "execution_count": 114,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 8 µs, sys: 1e+03 ns, total: 9 µs\n",
+      "Wall time: 12.2 µs\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": "32"
+     },
+     "execution_count": 114,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# write your numpy code here and estimate the time of execution"
+    "def unknown_signature(vector1, vector2):\n",
+    "    vector1 = np.array(vector1)\n",
+    "    vector2 = np.array(vector2)\n",
+    "    result = vector1 @ vector2\n",
+    "    return result\n",
+    "\n",
+    "vector1 = [1, 2, 3]\n",
+    "vector2 = [4, 5, 6]\n",
```

Comment:
Don't use python `list` in this task

Reply:
Why? I use it as a parametr

Reply:
The main gist of this task is to compare the work of python list and numpy array. You should rewrite the example with python list using numpy array. What is the sense of converting python list to numpy array if you can just use numpy array?

Reply:
I don't understand what I need to do...def unknown_signature(vector1, vector2):
    return vector1 @ vector2

vector1 = np.array([1, 2, 3])
vector2 = np.array([4, 5, 6])
result = unknown_signature(vector1, vector2)
Do u want it?